charts/flagsmith/values.yaml

```diff
@@ -14,6 +14,7 @@ api:
   # deployment's pods.
   separateApiAndFrontend: true
   replicacount: 1
+  deploymentStrategy: null
```

Comment:
Have we tested what happens if a custom strategy isn't defined? Does K8s still use the default RollingUpdate strategy?

Reply:
Seems like we've tested the case where we set the strategy (as per your comment above - "Deployed to minikube, tested that setting values for this do indeed get set on the deployments.") but just want to confirm we've tested the default case.

Reply:
For the default case, where you don't set a value for eg `api.deploymentStrategy` and let it stay as `null`, you get no change whatsoever in your template - the strategy remains unset, because that's how `with` works (see https://helm.sh/docs/chart_template_guide/control_structures/#modifying-scope-using-with, if the value is "empty", then the block doesn't execute). And of course because the template is the same, you get exactly the same behaviour as before, which is the k8s default of: RollingUpdate, maxUnavailable=25%, maxSurge=25%.